src/components/CatalogOverlay/CatalogPlotComponent/CatalogPlotComponent.tsx

```diff
@@ -611,6 +703,25 @@ export class CatalogPlotComponent extends React.Component<WidgetProps> {
             </FormGroup>
         );
 
+        const renderStatisticSelect = (
+            <FormGroup inline={true} label="Statistic Source">
```

Comment:
Statistic "Source" is hidden in the histogram panel. We could either set a larger default width or try to adjust the column widths
![image](https://user-images.githubusercontent.com/43841102/142819128-1a0848c9-86db-4be8-884f-8a045a1bfa52.png)